src/main/java/wenjigglybot/Parser.java

```diff
@@ -0,0 +1,138 @@
+package wenjigglybot;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Parser {
+
+    /**
+     * Processes the input string representing an event task.
+     * Extracts the event description, start time, and end time.
+     *
+     * @param task The input string containing the event details.
+     * @return A string array containing the event description, start time, and end time.
+     * @throws EventException If the input string does not contain the correct format.
+     */
+    public static String[] processEventTask(String task) throws EventException {
```

Comment:
Maybe change the name task to input since the type is String rather than a Task?